t/api/09-comments.t

```diff
@@ -156,6 +156,18 @@ subtest 'parent group comments' => sub {
     test_comments(parent_groups => 1);
 };
 
+subtest 'create many comments' => sub {
+    $t->post_ok('/api/v1/comments?job_id=42&job_id=foo')->status_is(400);
+    $t->json_is('/error' => 'Erroneous parameters (job_id invalid, text missing)');
+    $t->post_ok('/api/v1/comments?job_id=42&job_id=80000&text=batch-comment')->status_is(400);
+    $t->json_is('/error' => 'Not all comments could be created.', 'error returned');
+    $t->json_is('/failed' => [{job_id => 42}], 'failed IDs returned');
+    $t->json_is('/created' => [{job_id => 80000, id => 5}], 'created comment returned');
+    ok my $comment = $t->app->schema->resultset('Comments')->find(5), 'comment created' or return;
```

Comment:
that means if the "ok"-test did not pass then the code would return from here as continuing would lead to misleading errors when trying to work with non-existant comments.